frontend/src/Editor/Box.jsx

```diff
@@ -201,6 +200,7 @@ export const Box = function Box({
   const { variablesExposedForPreview, exposeToCodeHinter } = useContext(EditorContext) || {};
 
   const componentActions = useRef(new Set());
+  const darkMode = localStorage.getItem('darkMode') === 'true';
```

Comment:
I think this converted to function and be moved to a utility file for DRY purpose